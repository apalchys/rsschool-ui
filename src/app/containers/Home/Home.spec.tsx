import { shallow } from 'enzyme';
import * as React from 'react';
import Home from './index';
const { __stateGetter } = require('react-redux');

const noop = () => {};

describe('Home', () => {
    beforeEach(() => {
        __stateGetter.mockImplementation(() => ({}));
    });

    it('renders correctly if no data', () => {
        const output = shallow(<Home fetchFeed={noop} feed={[]} />);
        expect(output).toMatchSnapshot();
    });
});
