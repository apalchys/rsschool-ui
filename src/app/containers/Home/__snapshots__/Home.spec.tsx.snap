// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home renders correctly if no data 1`] = `
<Home
  feed={Array []}
  fetchFeed={[Function]}
/>
`;
